src/backend/distributed/deparser/deparse_table_stmts.c

```diff
@@ -103,10 +104,138 @@ AppendAlterTableStmt(StringInfo buf, AlterTableStmt *stmt)
 }
 
 
+/*
+ * AppendAlterTableCmdAddConstraint builds the add constraint command for index constraints
+ * in the ADD CONSTRAINT <conname> {PRIMARY KEY, UNIQUE, EXCLUSION} form and appends it to the buf.
+ */
+static void
+AppendAlterTableCmdAddConstraint(StringInfo buf, Constraint *constraint)
+{
+	/* Need to deparse the alter table constraint command only if we are adding a constraint name.*/
+	if (constraint->conname == NULL)
+	{
+		ereport(ERROR, (errmsg(
+							"Constraint name can not be NULL when deparsing the constraint.")));
```

Comment:
fyi: codeclimate says this uncovered

Reply:
That's because normally we shouldn't reach this